pkg/asset/installconfig/installconfig.go

```diff
@@ -42,32 +42,32 @@ var _ asset.WritableAsset = (*InstallConfig)(nil)
 func (a *InstallConfig) Dependencies() []asset.Asset {
 	return []asset.Asset{
 		&sshPublicKey{},
+		&platform{},
```

Comment:
i believe the reasoning was that instead of having architecture as the last query, it should up front as it is a basic platform specific setting. And since we support different arches for different platforms, the platform query needed to be moved up too.